src/RapidezServiceProvider.php

```diff
@@ -183,6 +185,8 @@ protected function bootTranslations(): self
     protected function bootListeners(): self
     {
         Event::listen(ProductViewEvent::class, ReportProductView::class);
+        MagentoSettingsHealthcheck::register();
+        ElasticsearchHealthcheck::register();
```

Comment:
Maybe keep this the same? I haven't seen the registration in the listener yet, is that a new thing?